### PR TITLE
Added Spanish QWERTY layout

### PR DESCRIPTION
**Pull request by margual56**
Added letter "ñ" to the layout and removed it from the popup. Also moved some of the accent keys closer to the beginning.

### CODE REVIEW

File: app/src/main/res/xml/keys_letters_spanish_qwerty.xml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="7"/>
+        <Key
+            app:keyLabel="i"
+            app:popupCharacters="8íìîïī"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="8"/>
+        <Key
+            app:keyLabel="o"
+            app:popupCharacters="ő9óôòõōö"
```
Comment (tibbi) [inline app/src/main/res/xml/keys_letters_spanish_qwerty.xml:47]: numbers should be the default values at long pressing a character

**Comment by tibbi** [comment]
this moves it actually even further from the default one. Are you compiling the app and testing at all?

**Comment by margual56** [comment]
> this moves it actually even further from the default one. Are you compiling the app and testing at all?

Sorry, my bad. This should fix it.

PS: Should I add the keyboard to the keyboards list?

**Comment by tibbi** [comment]
sure, you should do the whole thing

**Comment by margual56** [comment]
I hope that is correct :)

**Comment by tibbi** [comment]
will take a look at it later, thanks

**Pull request merged**